test/models/media_test.rb

```diff
@@ -71,7 +71,7 @@ class MediaTest < ActiveSupport::TestCase
     data = m.as_json
     assert_match 'https://www.bbc.com', m.url
     assert_match 'BBC', data['title']
-    assert_match /Breaking news/, data['description']
+    assert_match /Visit BBC|Breaking news/, data['description']
```

Comment:
I fixed this failing test here, although it is not related to this PR. The BBS uses a different `description` on their homepage in the US vs rest of the world, so this was failing when executed on Travis.

Reply:
A question: I had this same [issue](https://github.com/meedan/pender/pull/416/commits/58b8dbbd80cdf19436e9cf79a82b0615a932b5aa). I thought since we don't really have a BBC parser (it's standing for any generic url), testing for a string should be enough. So I updated it to be `assert_kind_of String, data['description']`. Do you think we should check for both descriptions or is checking for the string enough?

Reply:
Checking for any string is better. I prefer your approach, since the description can change at any time. I can revert this line so that yours can update it when it is merged, or would you rather I ported your fix to this MR?

Reply:
I still have to run at least one integration test, those take a while, so I think it's best to port the fix here and merge it.

Reply:
OK, thank you. 

I ported the fix.